### DISCUSSION TITLE
Question regarindg ram and disk

### DISCUSSION THREAD

**Discussion by manuelfickovic** [madMAx43v3r/chia-plotter#673] [Q&A]
Hi,
I have a question regarding RAM and disk.

If I have 384GB of ram on my machine, do I need to use SSD for plotting?

Right now I have 2 hard disks for destination and If I don't need SSD i can add 2hdd more.

Thanks in advance,
Manuel

**Comment by gryan315** [comment]
Directly from the usage section of the readme:

> tmpdir needs about 220 GiB space, it will handle about 25% of all writes. (Examples: './', '/mnt/tmp/')
> tmpdir2 needs about 110 GiB space and ideally is a RAM drive, it will handle about 75% of all writes.
> Combined (tmpdir + tmpdir2) peak disk usage is less than 256 GiB.

So yes, if you create a single large ramdisk, you can just run a tmpfs as -t
